chart/templates/ingress.yaml

```diff
@@ -6,12 +6,7 @@ metadata:
   name: {{ include "release" . }}
   namespace: {{ .Release.Namespace }}
 spec:
-  {{- if .Values.ingress.tls.enabled }}
-  tls:
-    - hosts:
-        - {{ .Values.hostname }}
-      secretName: {{ .Values.ingress.tls.secretName }}
-  {{- end}}
+  tls: {{ toYaml .Values.ingress.tls | nindent 4 }}
```

Comment:
`.values.ingress.tls` is not defined in env/dev.yaml. Is it OK to have an empty `tls:` section? or should we not include it instead?